cypress/integration/pages/errorPage404/index.js

```diff
@@ -1,6 +1,25 @@
 import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import tests from './tests';
-import testsForAMPOnly from './testsForAMPOnly';
-import testsForCanonicalOnly from './testsForCanonicalOnly';
+import { testsToAlwaysRun, tests, testsToNeverSmokeTest } from './tests';
+import {
+  testsToAlwaysRunForAMPOnly,
+  testsForAMPOnly,
+  testsToNeverSmokeTestForAMPOnly,
+} from './testsForAMPOnly';
+import {
+  testsToAlwaysRunForCanonicalOnly,
+  testsForCanonicalOnly,
+  testsToNeverSmokeTestForCanonicalOnly,
+} from './testsForCanonicalOnly';
 
-runTestsForPage('errorPage404', tests, testsForAMPOnly, testsForCanonicalOnly);
+runTestsForPage(
+  'errorPage404',
+  testsToAlwaysRun,
+  tests,
```

Comment:
Can't help but notice that all other tests have very descriptive names and this one is less obvious.

What about something like `testsSpecificToPageType`?

Reply:
Renamed, but not mentioned page type, because that's implicit in most locations the function is named.